.gitignore

```diff
@@ -1,5 +1,10 @@
 # -------------------- CUSTOM IGNORES -------------------
 
+# Python should ignore "lib" folders, however, flutter uses a "lib" folder that should not be ignored
+washee_web/**/lib/**
+washee_box/**/lib/**
```

Comment:
I added these, to make sure that the "lib" folders in python are correctly ignored (since they unlike Flutter needs to be ignored!)